OpenKeychain/src/main/java/org/sufficientlysecure/keychain/service/ContactSyncAdapterService.java

```diff
@@ -143,6 +143,13 @@ public IBinder onBind(Intent intent) {
     }
 
     public static void requestContactsSync() {
+        // if user has disabled automatic sync, do nothing
```

Comment:
for the record, a nicer way to do this is:
```
       boolean isSyncEnabled = ContentResolver.getSyncAutomatically(
                 new Account(Constants.ACCOUNT_NAME, Constants.ACCOUNT_TYPE), ContactsContract.AUTHORITY); 
       if (!isSyncEnabled) {
             return;
         }
```
it's the same number of lines, but instead of writing a comment you give the condition "isSyncEnabled" an explicit name. that way, someone who reads this can grasp everything that's happening by reading only java, without the need to parse english somewhere inbetween